ksp_plugin_adapter/reference_frame_selector.cs

```diff
@@ -641,7 +641,7 @@ private void RenderSubtree(CelestialBody celestial, int depth) {
               GUILayoutWidth(1))) {
         PlanetariumCamera.fetch.SetTarget(celestial);
         PlanetariumCamera.fetch.SetDistance(
-            (float)celestial.Radius * ScaledSpace.InverseScaleFactor * 2f);
+            (float)celestial.Radius * ScaledSpace.InverseScaleFactor * 2f * 4);
```

Comment:
Thanks for the change, which I agree will improve usability.  I would like it programmed a bit differently, as the factor 4 (or 8, really) looks a bit magic.
```suggestion
            (float)celestial.Radius * ScaledSpace.InverseScaleFactor /
                Math.Tan(apparent_size / 2));
```